app/frontend/src/components/AvatarInput.tsx

```diff
@@ -24,16 +26,23 @@ export function AvatarInput({
   src,
   username,
   className,
+  onChange,
   ...rest
 }: AvatarInputProps) {
   const classes = useStyles();
+  const [imageUrl, setImageUrl] = useState(src);
+  const handleChange = (values: ImageInputValues) => {
+    const randomInt = Math.floor(Math.random() * 100); // force reload onChange
+    setImageUrl(values.thumbnail_url + "?rand=" + randomInt);
+    onChange && onChange(values.key);
```

Comment:
```suggestion
    onChange?.(values.key);
```
You don't have to change this but I just learnt this recently and I think it's cool :D

Reply:
That's pretty cool actually 🤔

Reply:
This is unnecessary since `onChange` is guaranteed by the interface to always exist?

Reply:
```suggestion
    onChange(values.key);
```